.github/workflows/ruby.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
This file exists as https://github.com/dblock/strava-ruby-client/blob/master/.github/workflows/test.yml, delete this dup.